src/frontend/src/lib/components/networks/NetworkButton.svelte

```diff
@@ -28,7 +30,12 @@
 </script>
 
 <button class="w-full flex justify-between" on:click={onClick}>
-	<TextWithLogo {name} {description} {icon} logo="start" />
+	<TextWithLogo
+		{name}
+		{icon}
+		logo="start"
+		description={nonNullish(totalUsd) ? formatUSD(totalUsd) : undefined}
```

Comment:
I never remember: is this reactive? Meaning, if balance changes in the parent, it changes here to right?

Reply:
Yes, it re-renders if a prop changes